oliinykao/second/task3.py

```diff
@@ -0,0 +1,7 @@
+"""
```

Comment:
Rewrite in function:
`def last_letter_upper(string):
    list_of_words = re.split("[ .,!?:;-]", string)
    for word in list_of_words:
        print(word[:len(word)-1] + word[len(word) - 1].upper())
last_letter_upper(mytext)`

Reply:
Fixed